### ISSUE TITLE
can't connect to Talisman with showcase Dapp

### ISSUE THREAD

**Issue by arnobase** [TalismanSociety/talisman-connect#3]
On a fresh install of the showcase Dapp on my setup, I can't login with Talisman

When I click "Connect Wallet" and select "Talisman" I see a message with "Haven't got a wallet yet?" when I was expecting "Select Talisman account" 

![Capture d’écran 2022-02-11 à 11 25 05](https://user-images.githubusercontent.com/8094810/153592040-c7d254bb-0911-4b0d-84af-3780a0c294a0.png)

in the console, injectedWeb3 object contains only Polkadot.js (if I enable it in my browser)

Talisman extension is activated, 
app.talisman.xyz works as expected

![Capture d’écran 2022-02-11 à 11 26 42](https://user-images.githubusercontent.com/8094810/153592127-5769a2b0-3122-4539-a15c-df4ebbce7935.png)

Environement: 
os: MasOS 10.14
browser: Chrome 98 (same on Firefox 97)
Talisman alpha 0.9.1

step to reproduce :

install Talisman extension on the browser, add an accout
clone repository of [talisman-connect](https://github.com/TalismanSociety/talisman-connect)
yarn
yarn install
launch localhost:4200

**Comment by juliuscarvajal** [comment]
@arnobase 
The Talisman extension only works with https domains for security reasons.
There are steps to make it work on a local dev environment but would have to consult with the Talisman Extension team first.

Keeping this open for now.

**Comment by juliuscarvajal** [comment]
@arnobase 
**Steps to enable localhost Talisman Extension**

- Open manifest.json in the extension folder.
- Append `http://localhost:*/*` to wherever there’s `https://*//*`
Example:
```
  "content_scripts": [
    {
      "matches": ["https://*/*", "http://localhost:*/*"],
    }
  ],
  "permissions": ["https://*/*", "http://localhost:*/*", "storage", "tabs"],
```
- On chrome://extensions (or brave://extensions), reload the Talisman extension
- On your Dapp, refresh the browser

I will close this issue now but if this is still an issue for you, please reopen. Thanks.

**Comment by juliuscarvajal** [comment]
The extension team will be releasing an update that will enable the Talisman Extension on local dev by default. This should be fixed by `v0.10.0`.
